src/components/Clusters/ClusterDetail.tsx

```diff
@@ -1,5 +1,140 @@
+import { Button, Card, CardSection, Group, Text } from '@mantine/core';
+import { useState } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { RiDeleteBin5Line, RiSettings4Line, RiToggleFill, RiToggleLine } from 'react-icons/ri';
```

Comment:
Same thing try to find the equivalent icon in the /tb library. Else let's discuss about switching ri. I rather not mix though i.e some icons from ri and some from tb. Currently, we use only tb (except one font awesome icon which didn't have an equivalent in tb).